src/Raven.Client/Documents/Operations/AI/ChunkingOptions.cs

```diff
@@ -0,0 +1,29 @@
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.AI;
+
+public class ChunkingOptions : IDynamicJsonValueConvertible
+{
+    public ChunkingMethod ChunkingMethod { get; set; }
+
+    public int MaxTokensPerChunk { get; set; } = 2048;
```

Comment:
We should note that 2048 is pretty high - our embedded model only does 512 or so, IIRC.
And 512 looks like a very common value for BERT derived model.

512 tokens == ~2KB in chars, which is quite a bit. The question is how we define the default, and if we should set it to 512 as the common minimum?

At any rate, we rely on the _user_ to provide those for us, right? Should we provide the known values for the model during construction, instead?

Reply:
I think it should be something reasonable, especially not 0 as it was. Spoke with the guys and the told me a lot of 'remote' models has 2048, this is why we went with that. But we can go with 512, at the end I would expect user to set it to a desired number.

Reply:
Done